src/Simplex/Chat/Migrations/M20240402_item_forwarded.hs

```diff
@@ -0,0 +1,36 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Migrations.M20240402_item_forwarded where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20240402_item_forwarded :: Query
+m20240402_item_forwarded =
+  [sql|
+ALTER TABLE chat_items ADD COLUMN fwd_from_tag TEXT;
+ALTER TABLE chat_items ADD COLUMN fwd_from_chat_name TEXT;
+ALTER TABLE chat_items ADD COLUMN fwd_from_msg_dir INTEGER;
+ALTER TABLE chat_items ADD COLUMN fwd_from_contact_id INTEGER REFERENCES contacts ON DELETE SET NULL;
+ALTER TABLE chat_items ADD COLUMN fwd_from_group_id INTEGER REFERENCES groups ON DELETE SET NULL;
```

Comment:
As there are no joins here, were could just have one chat ID... No need to change.